pkg/cmd/server/origin/master.go

```diff
@@ -246,6 +251,9 @@ func (c *MasterConfig) Run(stopCh <-chan struct{}) error {
 	// add post-start hooks
 	aggregatedAPIServer.GenericAPIServer.AddPostStartHookOrDie("template.openshift.io-sharednamespace", c.ensureOpenShiftSharedResourcesNamespace)
 	aggregatedAPIServer.GenericAPIServer.AddPostStartHookOrDie("authorization.openshift.io-bootstrapclusterroles", bootstrappolicy.Policy().EnsureRBACPolicy())
+	aggregatedAPIServer.GenericAPIServer.AddPostStartHookOrDie("authorization.openshift.io-ensureSARolesDefault", ensureDefaultNamespaceServiceAccountRoles)
```

Comment:
the naming was strange before. `template.openshift.io` looks like an dns name. But then we have a `-something` postfix.

Reply:
> the naming was strange before. template.openshift.io looks like an dns name. But then we have a -something postfix.

The group so we know who to blame and then the particulars.  With separate servers, they reduce down to particulars.

Reply:
nvmd, cosmetics